src/main/java/org/saltyrtc/client/helpers/TaskHelper.java

```diff
@@ -36,15 +36,16 @@
      * Choose the first task in our own list of supported tasks that is also contained in the list
      * of supported tasks provided by the peer.
      *
-     * @return The selected task, if a common task can be found.
+     * @return The selected task if a common task can be found.
      */
-    public static Optional<Task> chooseCommonTask(Task[] ourTasks, List<String> theirTasks) {
+    @Nullable
+    public static Task chooseCommonTask(Task[] ourTasks, List<String> theirTasks) {
```

Comment:
Since we're touching this method, it makes sense to introduce `NonNull` for the arguments, too.